ambry-replication/src/main/java/com.github.ambry.replication/ReplicationManager.java

```diff
@@ -356,6 +356,8 @@ public void start()
         }
       }
 
+      replicationMetrics.trackLiveThreadsCount(replicaThreadPools, dataNodeId.getDatacenterName());
+
```

Comment:
can you move this to line 349. We need to know if something died immediately after starting.